themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -563,6 +563,8 @@ validating-public-key-id=Validating Public Key Id
 identity-provider.validating-public-key-id.tooltip=Explicit ID of the validating public key given above if the key ID. Leave blank if the key above should be used always, regardless of key ID specified by external IDP; set it if the key should only be used for verifying if key ID from external IDP matches.
 identity-provider.allowed-clock-skew=Allowed clock skew
 identity-provider.allowed-clock-skew.tooltip=Clock skew in seconds that is tolerated when validating identity provider tokens. Default value is zero.
+forwarded-query-parameters=Forwarded Query Parameters
+identity-provider.forwarded-query-parameters.tooltip=Query parameters to be forwarded to external IDP. Multiple parameters can be entered, separated by comma (,).
```

Comment:
I am not sure if this tooltip could be made more clear to specify from which place are parameters forwarded? For example something like: Query parameters to be forwarded to external IDP from the initial application request to Authorization Endpoint. Multiple parameters can be entered, separated by comma (,).